File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.py
class Solution:
    def getXORSum(self, arr1: List[int], arr2: List[int]) -> int:
        a, b = 0, 0
        
        for n in arr1:
            a ^= n
        
        for n in arr2:
            b ^= n
        
        return a & b

File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.cpp
class Solution {
public:
    int getXORSum(vector<int>& arr1, vector<int>& arr2) {
        return solve(arr1, arr2);
    }
private:
    // Aakash Sir
    int solve(vector<int>& arr1, vector<int>& arr2) {
        int a = 0, b = 0;
        
        for(int n: arr1)
            a ^= n;
        for(int n: arr2)
            b ^= n;
        
        return a & b;
    }
};